ste/folderCreationTracker.go

```diff
@@ -127,8 +203,8 @@ func (f *jpptFolderTracker) ShouldSetProperties(folder string, overwrite common.
 		defer f.mu.Unlock()
 
 		var created bool
-		if idx, ok := f.contents[folder]; ok {
-			created = f.plan.Transfer(idx).TransferStatus() == common.ETransferStatus.FolderCreated()
+		if idx, ok := f.contents.Get(folder); ok {
+			created = f.plan.Transfer(*idx).TransferStatus() == common.ETransferStatus.FolderCreated()
```

Comment:
We shall check for status created or success like the other check. Once creation is complete the status is set to success from created.